roles/elasticsearch/tasks/main.yml

```
@@ -12,7 +12,11 @@
   set_fact:
     elasticsearch_tls_key_passphrase: "{{ elastic_cert_pass }}"
   when:
-    - elastic_cert_pass is defined
+  - elastic_cert_pass is defined
```

```suggestion
    - elastic_cert_pass is defined
```
I guess 2 spaces more of indentation are needed here.